docker/web_nginx.conf

```diff
@@ -83,14 +91,27 @@ server {
         # app server is http only.
         proxy_set_header X-Scheme $scheme;
 
-        if ($http_user_agent ~ (Bytespider) ) {
-           return 444;
-        }
-
+    }
 
+    # Log likely bots caught in /authors random loop.
+    location ~* ^/authors/.* {
         # Web rate limit.
         limit_req zone=web_limit burst=200 nodelay;
         limit_req_status 429;
+
+        # randomly sorting will be removed. For now just return 200
+        if ($is_random_sort = 1) {
+            return 200 "";
```

Comment:
Note this is effectively indiscriminately returning 200 for all `/authors` requests with url param `sort=random_.*` . It would be nice if we could achieve this goal with a little less nginx config, but merging this as is for now because it's blocking deploys. (Took a look and was once again reminded the nginx configs are rather unpleasant to write :P )